BraveShared/Analytics/DAU.swift

```diff
@@ -213,21 +213,25 @@ public class DAU {
             return nil
         }
         
-        let dSecs = Int(today.timeIntervalSince1970) - stat[0]
+        let secondsSinceLastPing = Int(today.timeIntervalSince1970) - stat[0]
```

Comment:
Is this even used anymore? I think it is just used in the logging, right? If so, let's remove.

Reply:
Good catch, Xcode warning didn't show up because it's used in a debug log